src/components/DivElement.js

```diff
@@ -1,6 +1,13 @@
 const HTMLElement = require('./HTMLElement');
 
 // Define class here
-
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super();
```

Comment:
you have repetitive code in both constructors.   Instead of lines 6-9 you could have
super('div', content)